### PR TITLE
Add hpc IRC channel on Libera.chat

### PR DESCRIPTION
**Pull request by jwbowen**
Add links to the #hpc IRC channel on Libera.chat. While the #hpc channel is not affiliated with hpc.social (or anything else, for that matter), there is some overlap in membership.

### CODE REVIEW

File: _projects/chat.md [hunk 1]
```diff
@@ -37,3 +37,14 @@ If your preference is for Discord, we have a small community of HPC practitioner
...
+
+### IRC (#hpc on Libera.chat)
+
+<a href="https://web.libera.chat/#hpc" target="_blank"><button class="btn btn-primary btn_sm">Join the #hpc IRC channel on Libera.chat</button></a>
+
+If you empathize with <a href="https://xkcd.com/1782/" target="_blank">xkcd #1782 - Team Chat</a>,
+you're welcome to join the #hpc channel on <a href="https://libera.chat/" target="_blank">Libera.chat</a>.
+It's a small, laid back group of HPC practioners and friends who prefer the quaint world of IRC.
+
+> The #hpc IRC channel is not officially offiliated with hpc.social or any other group or project.
```
Comment (vsoch) [inline _projects/chat.md:49]:
```suggestion
> The #hpc IRC channel is not officially affiliated with hpc.social or any other group or project.
```

**Review by vsoch — COMMENTED** [review]
Super cool, and fantastic comic! One tiny typo and we should be good.

**Review by vsoch — APPROVED** [review]

**Comment by jwbowen** [comment]
Since the IRC channel isn't officially affiliated, I didn't feel it was appropriate to include it as one of the "Join hpc.social on..." buttons at the top.

**Comment by vsoch** [comment]
Totally OK and good by me! I’ll be sure to add this note when I share more broadly.

**Comment by jwbowen** [comment]
Sounds good. And thank you for reviewing this so quickly!

**Comment by vsoch** [comment]
My pleasure! I’m pretty much always around, which is both great and very, very sad 😂 😭

**Pull request merged**